.travis.yml

```diff
@@ -1,34 +0,0 @@
-language: python
-dist: bionic
-python:
-- '3.6'
-env:
-  global:
-  - DOCKER_USERNAME='ocstravisci'
-  - secure: Ho25dkqDdLbE0IRl2azR4Kucvw08gYjMRuuGUFQaG/iacwOdz/MLlhtLvEvsJPX1TBc3jIBggKxgq9CzcURQbQqPUWPjOdtzs2qiQ2W3eh7PTAbLz3arfAiIPt3kh5HyzLfPRj17S6a6HS1W1Gf7VPN0bXUW9I45zR6EmOtGsDiroWKEcR4azzmSioefrzBEDNSUTo+6hWONW4Fw3V/aAdHV/NTteFDyebDkosnrQqPdXPEYVUjfAIE3WDpyRH4tnZso+1+MDNSLyBnG4Y0mQil888LG4+rlqsUWlxrXpmQvyHrfz2lTzqmPOwo6hv7XHhwvAVCOPlS3XhF09Yma00QfSKU4FFGxAZ1yWn7bmY9eU5QuBXtCmBWpv/QoPU9hi4MjVFs9P35dqf/lj9HhtK4LzNcNpEis0LOe79OM/9w6NH1wIdUKLZwVKXdl1a62t1Rbt2yPqPfnfyz4XG03xFzIUVomq7Tjqgsg8fyMpRa0pFNLgMjWBtYDLtnooiyChondjoDWP6EbqZepj4hqdHfWjlhsHOImKHrvVzH/vMYbjL/idhJhpo0WTb/cYFN2m5Ijz3l2U1D5Gaa77xnqyVEOiEwHectyLcT+sbC4y3DSKvlyJwKHfp+bpVif29xfnQqw+89qrd4maXHiPeIWC7XnzU1jaEY91ko+WlzmdRg=
-addons:
-  apt:
-    packages:
-    - gfortran
-    - libblas-dev
-    - liblapack-dev
-services:
-- docker
-install:
-- pip install -U coverage coveralls
-- pip install 'numpy<1.20.0'
-- pip install ortools
-- mkdir logs
-- pip install -r requirements.pip
-script: nosetests --logging-level=WARNING test --exclude-dir=test/requires_third_party --with-coverage
-after_success: coveralls
-before_deploy:
-- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-- export IMAGE=observatorycontrolsystem/adaptive_scheduler:${TRAVIS_TAG}
-- docker build -t $IMAGE .
-deploy:
-- provider: script
-  script: docker push $IMAGE
-  skip_cleanup: true
-  on:
-    tags: true
```

Comment:
newline

Reply:
this file is being deleted, so the absence of a newline is going away

Reply:
lol i have baby brain